src/main/java/net/snowflake/ingest/connection/SecurityManager.java

```diff
@@ -154,14 +154,12 @@ private void regenerateToken() {
     // create our JWT claim builder object
     JWTClaimsSet.Builder builder = new JWTClaimsSet.Builder();
 
-    // set the subject to the fully qualified username
+    // get the subject to the fully qualified username
     String subject = String.format("%s.%s", account, user);
-    LOGGER.info("Creating Token with subject {}", subject);
 
-    // set the issuer
+    // get the issuer
     String publicKeyFPInJwt = calculatePublicKeyFp(keyPair);
```

Comment:
could 'calculatePublicKeyFp' ever fail? If so would we need the subject property in the log for debugging? If so, I can revert the changes and just set the log level to debug instead of info

Reply:
It can throw RTE, I would rather keep them two separate to have more information for debugging if we ever need that.